mod_test.ts

```diff
@@ -1126,3 +1192,55 @@ Deno.test("jsr specifiers - no lockfile", async (t) => {
     assertEquals(ns.join("a", "b"), join("a", "b"));
   });
 });
+
+Deno.test("ignores local css, uses default loader", async (t) => {
+  await testLoader(t, LOADERS, async (esbuild, loader) => {
+    if (esbuild === PLATFORMS.wasm) return;
```

Comment:
The default loaders do not appear to work at all with the wasm version of esbuild. You would need to use a plugin that has it's own loader for css or txt files for it to load them. That is why I have these 2 tests skipped for the wasm version of esbuild.